Source/WarpX.cpp

```diff
@@ -460,7 +460,10 @@ WarpX::ReadParameters ()
         pp.query("n_current_deposition_buffer", n_current_deposition_buffer);
         pp.query("sort_int", sort_int);
 
-        pp.query("quantum_xi", quantum_xi);
+        double quantum_xi;
+        int input_is_specified = pp.query("quantum_xi", quantum_xi);
```

Comment:
Maybe that's more specific as a name:
```suggestion
        int quantum_xi_is_specified = pp.query("quantum_xi", quantum_xi);
```